md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java

```diff
@@ -135,7 +135,7 @@ private void persistSingleBean(Object bean) throws IntrospectionException, Invoc
      */
     public void persistBeanData(Object bean, boolean printOnly) {
 
-        try {
+        /*try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)